WebContent/WOH-addRestaurant.jsp

```diff
@@ -0,0 +1,47 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+
+	<link rel="stylesheet" type="text/css" href="site/WOH-site.css">
+	<link rel="stylesheet" type="text/css" href="addRestaurant/WOH-addRestaurant.css">
+	
+	
+	<!-- Latest compiled and minified CSS -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+	
+	<!-- jQuery library -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+	<!-- Latest compiled JavaScript -->
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+	
+	<title>Wheel of Hunger - Add Restaurant</title>
+</head>
+
+<body>
+
+	<nav class="navbar navbar-inverse">
+  	<div class="container-fluid">
+    	<div class="navbar-header">
+      	<a class="navbar-brand" href="#">Wheel of Hunger</a>
```

Comment:
Change this href to "WOH-index.jsp".  I know it seems redundant but that way the anchor tag actually does something.  If you dont want to do that, change this anchor tag to a div.